haddock-api/src/Haddock.hs

```diff
@@ -1,6 +1,7 @@
 {-# OPTIONS_GHC -Wwarn #-}
 {-# LANGUAGE CPP, ScopedTypeVariables, OverloadedStrings, Rank2Types #-}
```

Comment:
nitpick, but can you split this line into several extensions, alphabetically-ordered? It's a tad weird to keep one big line and then have them be one per line

Reply:
(I realise GitHub doesn't show reviews on multiple lines, this is of course linked to line 4)